lib/overcommit/hook/pre_push/git_lfs.rb

```diff
@@ -0,0 +1,43 @@
+module Overcommit::Hook::PrePush
+  # Replaces the pre-push hook installed by `git lfs install` that is then moved aside by
+  # `overcommit --install`
+  #
+  # Enable this hook in .overcommit.yml like so:
+  #
+  #  PrePush:
+  #    GitLfs:
+  #      enabled: true
+  #      command: ['git', 'lfs', 'pre-push']
+  #
+  # @see https://github.com/github/git-lfs
+  class GitLfs < Base
+    # Overcommit expects you to override this method which will be called
+    # everytime your pre-commit hook is run.
+    def run
+      # Create two arrays to hold our error and warning messages.
+      error_lines = []
+      warning_lines = []
+
+      # Check if git-lfs executable exists
+      path_to_git_lfs = `command -v git-lfs`.strip
```

Comment:
Ya this had me scratching my head:
- Git Lfs installation will drop an executable (or symlink) named `/usr/local/bin/git-lfs`
- But their own pre_push hook executes `git lfs pre-push "$@"` which you can see [here](https://github.com/github/git-lfs/blob/8f90f594162d0a7e90e2f5785287f4327f097235/lfs/setup.go#L7)
So I'm left wondering, which should be the `required_executable`, `git` or `git-lfs`??